firewall/bypassing.go

```diff
@@ -15,7 +15,7 @@ func PreventBypassing(conn *network.Connection) (endpoints.EPResult, string, nsu
 	if strings.ToLower(conn.Entity.Domain) == "use-application-dns.net." {
 		return endpoints.Denied,
 			"blocked canary domain to prevent enabling of DNS-over-HTTPs",
-			nsutil.NxDomain("blocked canary domain to prevent enabling of DNS-over-HTTPs")
+			nsutil.NxDomain("")
```

Comment:
Was that intended? I though about something like:
```go
blockReason = "blocked canary domain to prevent ..."
return entpoins.Denied, blockReason, nsutil.NxDomain(blockReason)
```

Reply:
Yes, the reason is now always added to the response messages using the interface of the Connection itself. The argument to the responders is only for special cases within the nameserver.

Reply:
Maybe we should change it to `NxDomain(...string)` then so you don't need to pass in an empty string all the time?